contents/blog/best-mobile-app-session-replay-tools.mdx

```diff
@@ -0,0 +1,346 @@
+  <ComparisonRow column1={true} feature="Dead taps" description="Detect taps on elements that do not respond or trigger an action" />
+  <ComparisonRow column1={false} feature="Crash reports" description="Learn what happens before a crash without complex debugging or reproduction" />
+  <ComparisonRow column1={true} feature="AI summaries" description="Generate summaries of session replays" />
+</ComparisonTable>
+
+### How much does Microsoft Clarity cost?
+
+Clarity is completely free.
+
+### Why do companies use Microsoft Clarity?
+
+According to [reviews on G2](https://www.g2.com/products/microsoft-microsoft-clarity/reviews) and [Capterra](https://www.capterra.com/p/236349/Microsoft-Clarity/), it's because:
+
+1. **It's easy to set up and use:** Setting up Clarity doesn't require much technical knowledge and users mostly like the simple user interface.
+
+2. **It helps uncover how users navigate:** Clarity users are mostly using it to understand how people navigate their app and identify opportunities for improvement.
+
+3. **It's free:** The short 30-day retention limit is easy to forgive when it's free.
+
+> #### Bottom line
+> 
+> If you only care about session replay and dont need other features, Clarity is the obvious choice - especially if you're already using Google Analytics. On the other hand, software engineers will find Clarity's lack of debug logs disappointing.
+
+<br />
+
+## 3. PostHog
+
+**Typical users** Engineering and product teams
+  
+![posthog](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/screenshots/ab-testing.png)
+
+### What is PostHog?
+
+[PostHog](https://posthog.com) (that's us 👋) is an all-in-one platform built to help engineers create better products. It includes [session replays](/session-replay) **and** a whole bunch more, such as [product analytics](/analytics), [A/B testing](/ab-testing), and [feature flags](/feature-flags).
+
+It's designed for product-minded engineers, growth teams, and product managers who need to move fast and iterate based on reliable, actionable insights.
+
+#### Supported features
+
+<ComparisonTable column1="PostHog">
+  <ComparisonRow column1="iOS and Android only" feature="Session replay" description="Watch real users on your mobile apps; discover friction points" />
+  <ComparisonRow column1={true} feature="Debug logs" description="View console logs, network logs, performance data" />
+  <ComparisonRow column1={true} feature="Filters" description="View specific replays based on user segments, screens visited, events" />
+  <ComparisonRow column1={true} feature="Heatmaps" description="See where users tap and interact" />
+  <ComparisonRow column1={false} feature="Scroll maps" description="See how far users scroll and where they dwell" />
+  <ComparisonRow column1={true} feature="Rage taps" description="Identify repetitive taps made out of frustration" />
```

Comment:
Is this detected on the SDK side (JS SDK) or the backend based on all the click events?
If its on the backend, then we have it for free, otherwise, we don't have this on Mobile yet

Reply:
Maybe @daibhin knows, otherwise I can check once I am out of sick leave

Reply:
Nope, not within replay. We do have it in heatmaps but I don't know if that is relevant here